File: Python/challenge_3C.py
word = input("What is your word: ")
def vowels(word):
    vowels = "aeiou"

    number_vowels = 0
    for i in word:
        if i in vowels:
            number_vowels = number_vowels + 1
            print("number of vowels", number_vowels)
vowels(word)

File: Javascript/challenge3C.js

let vowels = "aeiou";
word = "greetings gurt"
function vowel_detector(vowels) {
    let number_vowels = 0
    for (let v of word) {
        if (vowels.includes(v)) {
            number_vowels = number_vowels + 1;
        }
    }
    console.log(number_vowels);
}
vowel_detector(vowels)